mediator/med_phases_aofluxes_mod.F90

```diff
@@ -968,16 +983,24 @@ subroutine med_aofluxes_update(gcomp, aoflux_in, aoflux_out, rc)
     integer               , intent(out)   :: rc
     !
     ! Local variables
-    type(InternalState)      :: is_local
-    integer                  :: n                          ! indices
-    real(r8), parameter      :: qmin = 1.0e-8_r8
-    real(r8), parameter      :: p0 = 100000.0_r8           ! reference pressure in Pa
-    real(r8), parameter      :: rcp = 0.286_r8             ! gas constant of air / specific heat capacity at a constant pressure
-    real(r8), parameter      :: rdair = 287.058_r8         ! dry air gas constant in J/K/kg
-    integer                  :: maptype
-    type(ESMF_Field)         :: field_src
-    type(ESMF_Field)         :: field_dst
-    character(*),parameter   :: subName = '(med_aofluxes_update) '
+    type(InternalState)    :: is_local
+    integer                :: n                          ! indices
+    real(r8), parameter    :: qmin = 1.0e-8_r8
+    real(r8), parameter    :: p0 = 100000.0_r8           ! reference pressure in Pa
```

Comment:
it would be nice if this was in med_constants_mod.F90

Reply:
That's a good point. I can put it there.